### PR TITLE
T248286: Jio2 'About the app' and 'About Wikipedia' layout fix

### PR DESCRIPTION
**Pull request by medied**
Phabricator Link: https://phabricator.wikimedia.org/T248286

### Problem Statement

'About the app' and 'About Wikipedia' pages are not rendering correctly in Jio2 devices.

### Solution

Minor CSS adjustments 

### Note

~To be tested on Jio2 device by someone who has one, I have requested this on the Phabricator thread. Let's not merge until it's confirmed it looks good on device~

Tested by Sudhanshu on Jio2 device now

### CODE REVIEW

File: style/aboutwikipedia.less [hunk 1]
```diff
@@ -44,6 +44,11 @@
 			font-weight: bold;
 			font-size: 22px;
 			line-height: 26px;
+
+			@media ( min-width: 250px ) {
```
Comment (hueitan) [inline style/aboutwikipedia.less]: what do you think we use it for

`@media @portrait` and `@media @landscape`

http://lesscss.org/#escaping

Comment (stephanebisson) [reply style/aboutwikipedia.less]: Using a variable does sound like a good idea to try to limit the spread of magic numbers.

Comment (medied) [reply style/aboutwikipedia.less]: Not a bad idea Huei, I introduced a `@landscape` variable now. Hoping that name won't be confusing (as these phones don't technically "support" landscape and adjust screen if a user turns it 90 degrees, right?).

The only other media query is a `min-width: 200px` used in `onboarding` and `aboutwikipedia`, does it make sense to set that as a `@portrait`variable?

Comment (hueitan) [reply style/aboutwikipedia.less]: `@portrait` is useful! 

Would be more clear if we add the comment to `variable.style` to said the dimension for portrait 240x320 and landscape 320x240?

**Comment by medied** [comment]
@jpita this is ready to be tested, note that [Sudhanshu did test](https://phabricator.wikimedia.org/T248286) what it looks like on Jio2 device.

**Pull request merged**